challenges/type-unions/prompt.md

```diff
@@ -0,0 +1,70 @@
+## What Problem Type Unions Solve
+
+Type In a previous example for `type-aliases` we saw a problem.  We were writing code for a space shuttle and we wanted to make sure that we can't accidentally use the wrong units type.
+
+```ts
+type Meters = number;
+type Miles = number;
+
+const landSpacecraft = (distance: Meters) {
+  // ... do fancy math ...
+}
+```
+
+But we noticed when we call `landSpacecraft` with the wrong literal type (see [the challenge on literal types](todo-link)), there's no error like we might expect:
```

Comment:
note: maybe we should use slugs instead of id so we can link easier

cc @Hacksore

Reply:
for SEO purposes you should strongly strongly strongly consider using slugs.  my intention was that these ids are also slugs.

also, number IDs like the type challenges incorrectly conveys the sense that there's some kind of ordering or something (which, there isn't.  it's a graph of challenges not an ordered list).

Reply:
will make it happen today